default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv
+, pkgs ? (import <nixpkgs> {})
+}:
+
+let
+  env = with pkgs; [
+    bundler
```

Comment:
An explanation would be nice, I can guess why it is there, but I want the official version.